oas_apivideo.yaml

```diff
@@ -13088,7 +13097,11 @@ paths:
       - name: viewDuration
         in: query
         description: | 
-          TODO
+          Use this query parameter to define how many seconds a view has to last to be counted in analytics data.
+
+          - You can only use this parameter together with the `view` metric.
```

Comment:
I'd remove the with maybe?
"You can only use this parameter with the `view` metric."
Or maybe "You can only use this parameter along the `view` metric." ?

(i'm forgetting how to English)

Reply:
Fixed: removed "together".